sql/rpl_group_replication.cc

```diff
@@ -103,12 +103,28 @@ int group_replication_init() {
     WSREP_ERROR(
         "Group replication cannot be used with Percona XtraDB Cluster in "
         "strict mode.");
+    if (current_thd) {
+      my_error(
+          ER_PXC_GR_INIT, MYF(0),
+          "Group replication cannot be used with Percona XtraDB Cluster in "
+          "strict mode.");
+    }
     return 1;
   } else {
     WSREP_WARN(
         "Using group replication with Percona XtraDB Cluster is only supported "
-        "for migration. Pease make sure that group replication is turned off "
+        "for migration. Please make sure that group replication is turned off "
```

Comment:
We have the same string message twice. It is probably better to use const char *msg ="...." and then use msg.

Reply:
fixed